src/utils/httpRequest.ts

```diff
@@ -4,5 +4,17 @@ class HttpRequest {
   static getMovies(query: string) {
     return axios.get(`http://localhost:4000/movies?${query}`);
   }
+
+  static deleteMovie(id: string) {
```

Comment:
Right now this service could be used only for one api, let's make it generic => 
let's get url and query parameters from outside and name methods as get/post/delete/put